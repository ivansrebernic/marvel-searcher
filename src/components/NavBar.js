import React from 'react'
import styled from 'styled-components'


const NavBarStyled = styled.nav`
    background-color:blue;
`

function NavBar(props) {


    return (
        <NavBarStyled>asd</NavBarStyled>
    )
}

export default NavBar