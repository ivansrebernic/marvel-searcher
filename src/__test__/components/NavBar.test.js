import React from 'react'
import { mount, shallow, render } from 'enzyme'
import NavBar from '../../components/NavBar'


describe('</NavBar>', () => {
    let wrapper
    beforeEach(() => {
        wrapper = shallow(<NavBar></NavBar>)
    });
    test('Render: </NavBar>', () => {
        expect(wrapper).toHaveLength(1)
    });
})