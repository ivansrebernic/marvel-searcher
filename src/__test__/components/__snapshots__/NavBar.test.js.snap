// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`</NavBar> Render: </NavBar> 1`] = `
<styled.nav>
  <styled.span />
  <SearchBox />
</styled.nav>
`;
